build.gradle.kts

```diff
@@ -173,6 +173,22 @@ subprojects {
     }
   }
 
+  apply(plugin = "com.diffplug.spotless")
+  configure<SpotlessExtension> {
```

Comment:
Will this fail the check build on unformatted PRs? Would be good to avoid general format rot.

Reply:
Yeah, check task already contains spotlessCheck.